module/purpose/Tester/data/config/Tester.yaml

```
@@ -8,7 +8,8 @@ tasks:
   find_ball_priority: 0
   look_at_ball_priority: 0
   walk_to_ball_priority: 0
-  walk_to_field_position_priority: 0
+  walk_to_kick_ball_priority: 0
+  walk_to_field_position_priority: 1
```

```suggestion
  walk_to_field_position_priority: 0
```
I assume all these should be zero by default